README.md

```diff
@@ -30,34 +31,40 @@
-- Highly customizable. You can modify everything displayed on the page with no special programming technique required.
-- Straightforward configuration. You may want to create your own version, and we have provided you with convenience. All you have to do is fill in the content you want, and our code will support you and do all the other work.
+ 
+- Loading fast. Most of static file can be distributed by [jsDelivr](https://jsdelivr.com), a global accelerated open source public CDN.
+- Highly customizable. You can modify everything display on the page with no special programming skills required.
+- Easy configuration. You may want to create your own version, and we have provided you with convenience. All you have to do is fill in the content you want, and our code will support you and do all the other work.
 - "Transformer" style. Your user may open your site to pay to you in a couple of different ways. `Merger` considered everything for you and it will present itself in different ways to make sure the user's need is fitted.
 
 ## Downloads
 
-- Please use the `release` function of this project at: [https://github.com/hifocus/merger/releases](https://github.com/hifocus/merger/releases). Please only use the latest, non pre-release version of `Merger` in production environment.
+- Please use the `release` function of this project at: [https://github.com/hifocus/merger/releases](https://github.com/hifocus/merger/releases). Please only use the `latest` non `pre-release version` of `Merger` in production environment.
 
-- If you have no idea about how to edit the `.html` file to get your own merged qrcode, we suggest you to use [Visual Studio Code](https://code.visualstudio.com/), an open source code editor.
+- If you have diffculties modifing the `.html` file to get your own merged qrcode, we suggest you to use [Visual Studio Code](https://code.visualstudio.com/), an open source code editor.
 Download it directly from HXCO Get: [https://get.soft.org/vscode](https://get.soft.org).
 
 ## Notices
 
 ### Bugs Notice
+
 During the development of this project, bugs might appear and affect your site. Here are two suggestions:
 1. Make sure your `Merger` version is at **stable**. To do this, download the latest release which is not `pre-release`. Stable version of `Merger` often contains latest security update and bug fixes. Stick with them to make sure nothing goes wrong.
 2. Check out frequently, or subscribe to our [Bugs Notice Board](https://github.com/hifocus/merger/issues/2). All knwon bugs will appear over there and their status will also be updated.
 
 ### ICP Notice for China
+
 In some Chinese app, especially for WeChat, the internal browser will display a security remind page for those domains that have no ICP record in China. Obviously, this is not friendly to users and they might doubt the safety of your site. Here are two solutions:
+
 1. Use **ICP recorded domain**, like our demo site: [https://demo.qrcdn.com](https://demo.qrcdn.com)
```

Comment:
I can't see any needs for putting blank lines here (they have no effect whether in vscode or github's markdown render). Especially you have ignored line 52.